.github/workflows/release.yml

```diff
@@ -0,0 +1,78 @@
+# This Workflow performs a full release of che-operator
+
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'version'
+        default: '7.x.y'
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+      - name: Install operator courier
+        run: |
+          python -m pip install --upgrade pip
+          # pip install operator-courier
+
+      - name: Release operator
+        run: |
+          CHE_VERSION=${{ github.event.inputs.version }}
+          BRANCH=${CHE_VERSION%.*}.x
+
+          #preinstall operator-courier and operator-sdk
+          pip install operator-courier==2.1.7
```

Comment:
should all the python related installations be done at the same time in the previous step?